_posts/en/newsletters/2019-01-22-newsletter.md

```diff
@@ -0,0 +1,111 @@
+---
+title: 'Bitcoin Optech Newsletter #30'
+permalink: /en/newsletters/2019/01/22/
+name: 2019-01-22-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a proposed LN feature to allow making
+spontaneous payments and provides our longest-ever list of notable code
+changes to popular Bitcoin infrastructure projects.
+
+## News
+
+- **PR opened for spontaneous LN payments:** LN protocol developer
+  Olaoluwa Osuntokun opened a [pull request][spontaneous payments] to
+  allow an LN node to send a payment to another node without first
+  receiving an invoice.  This takes advantage of LN's Tor-like onion
+  routing by allowing a spender to choose a preimage, encrypt it so
+  that only the receiver's node can decrypt it, and then route a payment
+  along LN like normal using the the hash of the preimage.  When the
+  payment reaches the receiver, they decrypt the preimage and disclose
+  it to the routing nodes in order to claim the payment.
+
+    Allowing spontaneous payments allows nodes to accept donations
+    without using extra software for generating invoices.  It can also
+    help in cases where users just want to do ad hoc payment
+    tracking, for example you initiate a 10 mBTC withdrawal from an
+    exchange and either 10 mBTC shows up in your balance within a few
+    moments or you contact support.  For serious payment tracking, users
```

Comment:
I don't much like the 'serious' here. Who's to say ad hoc payments aren't serious? Perhaps s/serious payment tracking/more significant payments/ or similar.